frontend/webpack.config.js

```diff
@@ -29,28 +51,15 @@ module.exports = {
 
   devServer: {
     historyApiFallback: true,
-    proxy: {
-      '/api': {
-        target: 'http://localhost:8000/',
-        secure: false
-      },
-      '/site-media': {
-        target: 'http://localhost:8000/',
-        secure: false
-      }
-    }
+    inline: true,
+    host: "0.0.0.0",
+    port: "8080"
+  },
+
+  watchOptions: {
+    poll: 100,
```

Comment:
@RyanNoelk Play around with these values. Apparently this is what fixes the refreshing issue on Mac.